packages/main/src/Select.ts

```diff
@@ -438,52 +445,53 @@ class Select extends UI5Element {
 			}
 		}
 
-		this._syncedOptions = syncOpts;
+		this._syncedOptions = syncOpts as Array<Option>;
```

Comment:
syncOpts is not array of Option (as Nayden also said in his review), although they share similar props, the Option extends Ui5 Element, HTMLElement and so on. So this is wrong in that regards. We need to declare a new type.

I saw it's used for the entries in the List in the Select Popover, so it seems that these fields on the syncedOptions are the things required for an option to be properly rendered as a list item. So it could be also `interface IOption`, and the Option component itself can implement it.